### ISSUE TITLE
FEATURE: List migrations

### ISSUE THREAD

**Issue by marchy** [seclerp/rider-efcore#134]
Now that the wonderful console view is implemented, could you please add the command to list migrations?

ie: Having a new 'List Migrations' command here:
<img width="168" alt="Screenshot 2022-12-10 at 6 25 51 PM" src="https://user-images.githubusercontent.com/1369360/206879277-6d292123-354c-41e2-aaa1-9ee640eb399f.png">

This is really crucial when working across multiple environments and you want to check which migrations are currently applied – as well as during development as you temporarily up and down migrations until you get the ORM mappings in the DbContext set up just right.

**EF command**
`dotnet ef migrations list`

_NOTE: The same parameters from the other commands should be included, such as the starting project, custom arguments being passed in etc._

## Keep rocking! 💪

**Comment by seclerp** [comment]
Yeah, I think this is quite possible to do. Which particular result representation do you expect after command execution? I think there should be 2 of them:
- If "Execute commands in terminal" is **enabled**: show output in that window
- If "Execute commands in terminal" is **disabled**: show output in a newly opened modal dialog right after execution is finished

**Comment by marchy** [comment]
Hmm I wasn’t really considering the scenario where the terminal window was disabled - maybe can keep it easy and require that it’s enabled?

Otherwise can run into scenarios where folks have dozens (or even hundreds?) of migrations applied and would need a more complex scrolling dialog.

**Comment by marchy** [comment]
Is this by chance set to make it in one of the upcoming releases?

**Comment by seclerp** [comment]
Hi @marchy, It's possible but I don't have a specific ETA right now, will update the milestone when it will be cleaner.

**Comment by marchy** [comment]
Thank you, kindly appreciate the update!  🙏
